### DISCUSSION TITLE
Using .chezmoiignore with different operating systems

### DISCUSSION THREAD

**Discussion by cadenkriese** [twpayne/chezmoi#2291] [Q&A] [answered]
My .chezmoiignore currently looks like this,
```
readme.md
scoopfile.json
Brewfile
dot_config/neofetch/m1-max.png

{{ if eq .chezmoi.os "windows" }}
dot_config/iterm2/*
dot_zshrc
empty_dot_hushlogin
{{ end }}
```
When running `cat .chezmoiignore | chezmoi execute-template` it shows what you would expect,
```
readme.md
scoopfile.json
Brewfile
dot_config/neofetch/m1-max.png


dot_config/iterm2/*
dot_zshrc
empty_dot_hushlogin
```
However, when running `chezmoi ignored` the conditional section is missing.
```
Brewfile
readme.md
scoopfile.json
````

I appreciate any insight!

**Comment by twpayne** [comment, marked as answer]
`.chezmoiignore` uses filenames in the target state, not the source state, so that it is not affected by changes to attributes. Your `.chezmoiignore` file should contain:

```
readme.md
scoopfile.json
Brewfile
.config/neofetch/m1-max.png

{{ if eq .chezmoi.os "windows" }}
.config/iterm2/*
.zshrc
.hushlogin
{{ end }}
```
